tcms/report/views.py

```diff
@@ -82,11 +83,15 @@ def overview(request, product_id, template_name='report/overview.html'):
 class ProductVersionReport(TemplateView, ProductVersionReportData):
     template_name = 'report/version.html'
 
-    def get(self, request, product_id):
+    def __init__(self):
+        self.product = None
+        self.selected_version = None
```

Comment:
I see this repeating a few times. See if writing it as class attributes under `template_name` will make pylint happy. There isn't a big difference between the two syntaxes but the last one is a bit shorter and will produce cleaner diff.